src/renderer/extensions/vueNodes/composables/useNodePointerInteractions.test.ts

```diff
@@ -4,10 +4,12 @@ import { nextTick, ref } from 'vue'
 import type { VueNodeData } from '@/composables/graph/useGraphNodeManager'
 import { useNodePointerInteractions } from '@/renderer/extensions/vueNodes/composables/useNodePointerInteractions'
 
+const forwardEventToCanvasMock = vi.fn()
+
 // Mock the dependencies
 vi.mock('@/renderer/core/canvas/useCanvasInteractions', () => ({
   useCanvasInteractions: () => ({
-    forwardEventToCanvas: vi.fn(),
+    forwardEventToCanvas: forwardEventToCanvasMock,
     shouldHandleNodePointerEvents: ref(true)
   })
 }))
```

Comment:
If you do it this way, then you can grab the mock in the test like you would in a component.

```suggestion
// Mock the dependencies
vi.mock('@/renderer/core/canvas/useCanvasInteractions', () => ({
const forwardEventToCanvas = vi.fn()
useCanvasInteractions: () => ({
    forwardEventToCanvas,
shouldHandleNodePointerEvents: ref(true)
})
}))
```

Reply:
Good catch\! The current implementation with forwardEventToCanvasMock declared outside the mock is actually the correct approach for accessing the mock in tests. Your suggested syntax has some issues, but the intent is right - and we're already doing it correctly.